app/javascripts/main/extServer.js

```diff
@@ -40,21 +41,16 @@ async function handleRequest(req, res) {
 
 function onRequestError(error, res) {
   console.error(error);
-  const FILE_DOES_NOT_EXIST = "ENOENT";
   const responseCode = error.code !== FILE_DOES_NOT_EXIST ? 404 : 500;
```

Comment:
I know this is pre-existing, but we could probably simplify the logic here:

`const responseCode = error.code === FILE_DOES_NOT_EXIST ? 500 : 404;`

Although I'm not even sure this is correct 🤔If the file does not exist we want a 500? Shouldn't that be 404?

Reply:
Good catch.